src/LodeRunner.API/src/Controllers/TestRunsController.cs

```diff
@@ -164,7 +195,7 @@ public async Task<ActionResult> DeleteTestRun([FromRoute, SwaggerRequestBody("Th
 
             if (existingTestRunResp.StatusCode == HttpStatusCode.OK)
             {
-                if (existingTestRunResp.Model.CompletedTime != null)
+                if (existingTestRunResp.Model.CompletedTime != null || (existingTestRunResp.Model.CompletedTime == null && existingTestRunResp.Model.StartTime > DateTime.UtcNow))

```

Comment:
```suggestion
                if (existingTestRunResp.Model.CompletedTime != null || (existingTestRunResp.Model.CompletedTime == null && existingTestRunResp.Model.StartTime >= DateTime.UtcNow))

```

Reply:
What are your thoughts on making the start time condition greater than or equal to now?